index.js

```diff
@@ -51,6 +51,12 @@ Plugin.prototype.apply = function(compiler) {
       }
 
       var chunks = {};
+      var outputDir = this.options.path || '.';
+      var outputFilename = path.join(outputDir, this.options.filename || DEFAULT_OUTPUT_FILENAME);
+      var obj = JSON.parse(fs.readFileSync(outputFilename, 'utf8'));
+      if (obj.status == 'done') {
+          chunks = obj.chunks;
+      }
```

Comment:
You can clone this repo to reproduce it: https://github.com/JohnnyZhao/bundle-tracker-reproduce